docs/core-concepts/model-basics.md

```diff
@@ -278,6 +278,35 @@ Foo.init({ /* attributes */ }, {
 });
 ```
 
+## Prevent creating a default PK attribute
+
+By default, Sequelize automatically adds the primary key field `id` to every model when there is no primary key field defined. When there is an `id` field but not a primary key field Sequelize will throw an error. To prevent this you can use the option `noPrimaryKey` when creating the instance and set it to true.
```

Comment:
A small change because I would prefer to avoid using the word `field` and instead always refer to it as either `column` (when talking about sql) or `attribute` (when talking about js)

I would also propose to remove the part about sequelize throwing an error if `id` is added manually, but its `primaryKey` option is not configured for a few reasons:

- it's unrelated to `noPrimaryKey` (it works as a "hey, did you mean that?")
- the error it throws is clear enough
- it's not necessary for the user to know about it until they encounter the error, at which point the error will explain what is going on. They're never going to actively seek this information.

```suggestion
By default, Sequelize automatically adds the primary key attribute `id` to every model when there is no primary key has been defined manually. To prevent this you can use the option `noPrimaryKey` when creating the instance and set it to true.
```

Reply:
> A small change because I would prefer to avoid using the word `field` and instead always refer to it as either `column` (when talking about sql) or `attribute` (when talking about js)
> 
> I would also propose to remove the part about sequelize throwing an error if `id` is added manually, but its `primaryKey` option is not configured for a few reasons:
> 
> * it's unrelated to `noPrimaryKey` (it works as a "hey, did you mean that?")
> * the error it throws is clear enough
> * it's not necessary for the user to know about it until they encounter the error, at which point the error will explain what is going on. They're never going to actively seek this information.

Interesting explanation 😄 , I will continue with resolving the other tomorrow.